src/container.ts

```diff
@@ -609,6 +609,17 @@ export class Container {
       throw new Error('Attempted to construct a container from a Container object.');
     }
 
+    const defaultResourceSpec: ContainerResources = {
```

Comment:
These values need to be documented in the `resources` property of `ContainerProps`.

Reply:
Will add them.